src/block/accordion/index.php

```diff
@@ -66,3 +66,64 @@ function stackable_load_accordion_frontend_polyfill_script() {
 	}
 	add_action( 'stackable/accordion/enqueue_scripts', 'stackable_load_accordion_frontend_polyfill_script' );
 }
+
+
+if ( ! function_exists( 'stackable_render_block_accordion_FAQ_schema' ) ) {
+	function get_answer( $block, $answer ) {
```

Comment:
The `get_answer` function name is globally available. To prevent conflicts and overrides, use a namespaced function name like `stackable_faq_schema_get_answer`

```suggestion
	function get_answer( $block, $answer ) {
```